apeunit/even-music
# #002 Feat: Set Up Nextjs and Tailwindcss

# PR
## PR Description
set up the nextjs and add tailwind css to the project 

## Task ID and Ticket Card Link
Task ID: # 002

Ticket Card Link: [here](https://www.notion.so/apeunit/Set-up-NextJS-and-TailwindCSS-3e69b11f72254dcbbfb5b74ea564c586)

## Completed Tasks
- [x] add nextjs
- [x] add tailwind css
- [x] remove unnecessary files

## Testing
- run npm install
-  npm run dev
- check if tailwind is working in the browser
